pocs/observatory.py

```diff
@@ -59,8 +58,10 @@ def __init__(self, cameras=None, *args, **kwargs):
         self.dome = pocs.dome.create_dome_from_config(self.config, logger=self.logger)
 
         self.logger.info('\tSetting up scheduler')
-        self.scheduler = None
-        self._create_scheduler()
+        self.scheduler = scheduler
+        if self.scheduler is None:
+            self.scheduler = pocs.scheduler.create_scheduler_from_config(self.config, logger=self.logger,
```

Comment:
We don't want to automatically create a scheduler if one is not passed during construction. Probably nothing needs to happen here, except maybe a log message stating that there is no scheduler.